modules/querying/pages/func/vertex-methods.adoc

```diff
@@ -941,26 +1027,32 @@ RUN QUERY selectVertex("/home/tigergraph/mydata/selectVertex.csv")
 === `to_vertex()`
 
 [WARNING]
-====
+[discrete]
```

Comment:
Have you verified that this is okay? I have a feeling that the warning admonishing will not be displayed correctly